### DISCUSSION TITLE
Publii on multiple Clients

### DISCUSSION THREAD

**Discussion by CLOUDCOOKCH** [GetPublii/Publii#1866] [General]
Hi there
I just wanted to ask if its possible to use publii on multiple clients for the same homepage?
As i am Switching between Desktop and Laptop alot that would be very cool.
Is there an easy way to do that?

**Comment by treeandcoffee** [comment]
Hi 
One possibility would be this: [Accessing your Publii site on multiple computers via Dropbox](https://getpublii.com/docs/publii-on-multiple-computers-via-dropbox.html)

**Comment by michaelfluegge** [comment]
this was asked several times in this forum. Don't be shy, try a search
